docs/serving/samples/hello-world/helloworld-csharp/helloworld-csharp.csproj

```diff
@@ -1,15 +1,14 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netcoreapp2.2</TargetFramework>

```

Comment:
Updating the `TargetFramework` to 2.2 is probably a good idea. I didn't notice this with my PR because I was only working on the Dockerfile.

Should decide whether these Dockerfile PRs are a good place to update this.